### PR TITLE
Adding GitHub Workflows to create tag and release Docker image

### PR DESCRIPTION
**Pull request by jtyr**
This PR is adding two GitHub Workflows that automatically create Git tag and publish Docker image on every merged PR. The Docker image is pushed into the GitHub Container Registry (GHCR) and will be visible in the repo as a [Package](https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages).

### CODE REVIEW

File: .github/workflows/tag.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+jobs:
+  tag:
+    name: Tagging
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```
Comment (gr2m) [inline .github/workflows/tag.yaml:27]: What scopes does the `PERSONAL_ACCESS_TOKEN` token need? I assume only `public_repo`?

Can we use `secrets.GITHUB_TOKEN` here instead?

Comment (jtyr) [reply .github/workflows/tag.yaml]: Sorry, that was a left over from where I copied it. We don't need any extra token for that. I have removed it in the last commit.

**Review by gr2m — APPROVED** [review]
Thanks!

**Comment by jtyr** [comment]
@gr2m Please could you create a new tag (e.g. `v1.0.0`) so we can see if the Docker image gets created?

**Comment by gr2m** [comment]
Done

**Pull request merged**